components/SortControl.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styles from '../styles/SortControl.module.css';
+
+export interface SortControlProps {
+    onSelectedSortChange: (newValue: number) => void;
+}
+
+export default function SortControl(props: SortControlProps) {
+    const onSelectedSortChange = React.useCallback(
+        (cb: React.ChangeEvent<HTMLSelectElement>) => {
+            props.onSelectedSortChange(parseInt(cb.currentTarget.value));
+        },
+        [props.onSelectedSortChange]
+    );
+
+    return (
+        <>
+            <div className={styles.sortSelector}>
+                <label htmlFor="sort-select">Sort By</label>
+                <select name="sort" id="sort-select" onChange={onSelectedSortChange}>
+                    <option value="0" key="points">Points</option>
```

Comment:
Good start. However, too much manual work if we were to add a third sorting option. Would instead like to see the list of potential sorting methods taken in as a prop instead of being hard defined.